packages/kununu-footer/index.scss

```diff
@@ -190,13 +215,13 @@
 }
 
 .infoTextColumn {
+  width: 100%;
+  text-align: center;
+
   @media (min-width: $screen-md-min) {
+    text-align: right;
     max-width: 550px;
   }
-
-  @media (max-width: $screen-sm-max) {
-    width: 100%;
-  }
 }
 
 div.title {
```

Comment:
Do we still need this rule?

Reply:
Hmm yes, but not here. I fixed it! Thanks for catching it 👍